<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="orderMenu">
    <property name="title">
     <string>Order</string>
    </property>
    <addaction name="new_order"/>
    <addaction name="delete_order"/>
    <addaction name="update_order_status"/>
    <addaction name="show_order_status"/>
   </widget>
   <widget class="QMenu" name="viewMenu">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="show_menu"/>
    <addaction name="show_kitchen"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="about_button"/>
   </widget>
   <widget class="QMenu" name="menuApplication">
    <property name="title">
     <string>Application</string>
    </property>
    <addaction name="show_todays_reports"/>
    <addaction name="quit"/>
   </widget>
   <addaction name="menuApplication"/>
   <addaction name="orderMenu"/>
   <addaction name="viewMenu"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="new_order">
   <property name="text">
    <string>New Order</string>
   </property>
  </action>
  <action name="delete_order">
   <property name="text">
    <string>Delete Order</string>
   </property>
  </action>
  <action name="update_order_status">
   <property name="text">
    <string>Update Order Status</string>
   </property>
  </action>
  <action name="show_order_status">
   <property name="text">
    <string>Show Order Status</string>
   </property>
  </action>
  <action name="show_menu">
   <property name="text">
    <string>Menu</string>
   </property>
  </action>
  <action name="show_kitchen">
   <property name="text">
    <string>Kitchen</string>
   </property>
  </action>
  <action name="menu_view">
   <property name="text">
    <string>Menu</string>
   </property>
  </action>
  <action name="actionAbout_this_application">
   <property name="text">
    <string>About This Application</string>
   </property>
  </action>
  <action name="show_todays_reports">
   <property name="text">
    <string>Show Today's Report</string>
   </property>
  </action>
  <action name="quit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="show_order_queues">
   <property name="text">
    <string>Show Order Queues</string>
   </property>
  </action>
  <action name="about_button">
   <property name="text">
    <string>About this app</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
